gemrb/plugins/AREImporter/AREImporter.cpp

```diff
@@ -1575,13 +1575,33 @@ void AREImporter::AdjustPSTFlags(AreaAnimation *areaAnim) {
 	 * break things (like stopping early, hiding under FoW).
 	 *
 	 * So far, a better approximation towards handling animations is:
-	 * - always set A_ANI_BLEND, A_ANI_SYNC,
-	 * - unset A_ANI_PLAYONCE, A_ANI_NOT_IN_FOG, A_ANI_BACKGROUND.
+	 * - zero everything
+	 * - always set A_ANI_SYNC
+	 * - copy/map known flags (A_ANI_ACTIVE, A_ANI_NO_WALL, A_ANI_BLEND)
+	 *
+	 * Note that WF_COVERANIMS is enabled by default for PST, so ANI_NO_WALL
+	 *   is important.
 	 *
 	 * The actual use of bits in PST map anims isn't fully solved here.
 	 */
-	areaAnim->Flags |= (A_ANI_BLEND | A_ANI_SYNC);
-	areaAnim->Flags &= ~(A_ANI_PLAYONCE | A_ANI_NOT_IN_FOG | A_ANI_BACKGROUND);
+
+	#define PST_ANI_ACTIVE 0x0001
+	#define PST_ANI_NO_WALL 0x0008
+	#define PST_ANI_BLEND 0x0100
```

Comment:
How about something like this:
```suggestion
	#define PST_ANI_NO_WALL 0x0008 // A_ANI_PLAYONCE in other games
	#define PST_ANI_BLEND 0x0100 // A_ANI_BACKGROUND in other games
```
Documents the mismatch, while PST_ANI_ACTIVE is the same as A_ANI_ACTIVE, so there's no need for it.